src/screen/dashboard.rs

```diff
@@ -64,7 +64,12 @@ impl Dashboard {
     pub fn restore(dashboard: data::Dashboard) -> (Self, Command<Message>) {
         let mut dashboard = Dashboard::from(dashboard);
 
-        let command = dashboard.track();
+        let command;
+        if let Some((pane, _)) = dashboard.panes.panes.iter().next() {
+            command = Command::batch(vec![dashboard.focus_pane(*pane),dashboard.track()]);
```

Comment:
Missing space after comma

Reply:
New to Rust and forgot to run the modified code through rustfmt.  Fixed now.  Thank you for the review.